CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 PROJECT (libsolv)
 
 CMAKE_MINIMUM_REQUIRED (VERSION 2.4)
+IF (COMMAND CMAKE_POLICY)
+CMAKE_POLICY(SET CMP0003 NEW)
```

Comment:
why?

Reply:
If I don't, I get the following:
```
CMake Warning (dev) at ext/CMakeLists.txt:137 (ADD_LIBRARY):
  Policy CMP0003 should be set before this line.  Add code such as

    if(COMMAND cmake_policy)
      cmake_policy(SET CMP0003 NEW)
    endif(COMMAND cmake_policy)

  as early as possible but after the most recent call to
  cmake_minimum_required or cmake_policy(VERSION).  This warning appears
  because target "libsolvext" links to some libraries for which the linker
  must search:

    zck

  and other libraries with known full path:

    /home/jonathan/Documents/programming/libsolv/build/src/libsolv.so.0

  CMake is adding directories in the second list to the linker search path in
  case they are needed to find libraries from the first list (for backwards
  compatibility with CMake 2.4).  Set policy CMP0003 to OLD or NEW to enable
  or disable this behavior explicitly.  Run "cmake --help-policy CMP0003" for
  more information.
This warning is for project developers.  Use -Wno-dev to suppress it.
```

Reply:
I think this might indicate wrong pkg-config file for zck, however I can't test it since zck is not packaged for Fedora.

Reply:
There is a copr at https://copr.fedorainfracloud.org/coprs/jdieter/zchunk/, but, you're right, we really need to get this into Fedora.

Neal, are you up for a package review?

Reply:
Just assign me for review.

Reply:
Great, thank you!

Reply:
I'm ready to review as well.

Reply:
Ok, Igor, I've assigned it to you, and, Neal, I've cc'd you into the bug.  Thanks so much, both of you.

Reply:
I don't see any bugs assigned to i.gnatenko.brain@gmail.com

Reply:
Sorry, I assigned it to ignatenko@redhat.com since that's what it looked like your bugzilla account was.  I'll re-assign it.